### ISSUE TITLE
We at Sandstorm like your app!

### ISSUE THREAD

**Issue by neynah** [dburles/treel#1]
Hi!

I'm the designer at [sandstorm.io](http://sandstorm.io) and tried out your app! I thought it was pretty cool and that it'd be a great fit for Sandstorm-I would personally use it myself. We're currently looking for unique open-source apps to add to our [App Market](https://apps.sandstorm.io/) and I think our users would definitely want to use treel.

Would you possibly be interested in helping me get your app running on Sandstorm so that others can quickly install & create instances of treel? I was able to package it successfully but perhaps you'd like to [test it out on Sandstorm](https://oasis.sandstorm.io/appdemo/pvrdjhwtx8n9dryez67tmrj9fm0067xdrxv8quqgvjtx7a1gw5w0) to see if it works as you want it to.

I'm pretty excited about this so if you decide you are interested in polishing this, I'd be happy to design some icons/graphics for you.

Cheers! :)

**Comment by dburles** [comment]
Hey @neynah sounds cool to me, I haven't updated the app in a while but I'd like to give it a look over before launching it.

**Comment by neynah** [comment]
Hi @dburles that's awesome news! You can checkout the existing demo of treel [here](https://oasis.sandstorm.io/appdemo/pvrdjhwtx8n9dryez67tmrj9fm0067xdrxv8quqgvjtx7a1gw5w0). Feel free to revise your app in your repo and let me know once it's ready and I'll package it again for your review.

In the meantime, maybe I can start on graphics? Perhaps you need a logo or some icons?

**Comment by dburles** [comment]
Yeah a logo would be nice, I had come up with one rather quickly which you can see top right on the login/register screen, (here's a screenshot). I actually quite like it, maybe we can work with that idea? If the integration with sandstorm necessitates some icons then I'm happy for that to be done too.

<img width="90" alt="screen shot 2015-11-05 at 12 18 41 pm" src="https://github-cloud.s3.amazonaws.com/assets%2F319567%2F10957339%2F70d643a8-83b7-11e5-9e0f-1a093ce96c91.png">

**Comment by neynah** [comment]
Awesome! How about having 2 tones (similar to your app UI). & Yes, I can put together the icons for you. We need 3, one for the app market, another for the app install grid, and another for the mini icon on the sidebar.
![treel-mockup](https://github-cloud.s3.amazonaws.com/assets%2F855341%2F10957521%2Fe8618e3a-8319-11e5-97a8-3c86b54a5149.png)

**Comment by neynah** [comment]
Hi @dburles, how's your app coming along? I'd love to help you get this on our App Market. :)

**Comment by dburles** [comment]
Hey sorry for the lack of progress @neynah I would definitely still like to get the source code up to date since this project has a bit of age to it, I'll ping you here once I make a move on it :)
